config/modules/cloudonly.conf

```diff
@@ -0,0 +1,5 @@
+# Cloud only tests
+
+build ${DAQ_LIB}/docker/modules
+include ${DAQ_LIB}/usi/build.conf
```

Comment:
Why USI?  That's not cloud oriented... ?  Is it because there's some part of the system that always tries to run the USI test, even if not explicitly enabled?

Also, can you just name the file "cloud.conf" -- just keeping in the minimalist spirit of the other files in that directory...

Reply:
Re USI - yes, I'd assumed with switch control being abstracted it was the module used for controlling the virtual switch so assumed it's needed. I'll test without it today and omit if not required